client/gefyra/__main__.py

```diff
@@ -250,6 +250,20 @@ def telemetry_command(on, off):
         logger.info("Invalid flags. Please use either --off or --on.")
 
 
+def prepare_cargo_endpoint(endpoint: str):
+    delimiter_count = endpoint.count(":")
+    if delimiter_count == 0:
+        return f"{endpoint}:31820"
+    elif delimiter_count == 1:
+        if endpoint[-1] == ":":
+            return f"{endpoint}:31820"
```

Comment:
Is this the case for input `127.0.0.1:`? If yes, this would become `127.0.0.1::31820`. This is probably not intended.